src/AmqpBundle/Sandbox/NullEnvelope.php

```diff
@@ -0,0 +1,358 @@
+<?php
+
+namespace M6Web\Bundle\AmqpBundle\Sandbox;
+
+/**
+ * Envelope being created from array
+ */
+class NullEnvelope extends \AMQPEnvelope
+{
+    /**
+     * @var string
+     */
+    private $body;
+    /**
+     * @var string
+     */
+    private $routingKey;
+    /**
+     * @var string
+     */
+    private $deliveryTag;
+    /**
+     * @var string
+     */
+    private $deliveryMode;
+    /**
+     * @var bool
+     */
+    private $redelivery;
+    /**
+     * @var string
+     */
+    private $contentType;
+    /**
+     * @var string
+     */
+    private $contentEncoding;
+    /**
+     * @var string
+     */
+    private $type;
+    /**
+     * @var string
+     */
+    private $timestamp;
+    /**
+     * @var int
+     */
+    private $priority;
+    /**
+     * @var string
+     */
+    private $expiration;
+    /**
+     * @var string
+     */
+    private $userId;
+    /**
+     * @var string
+     */
+    private $appId;
+    /**
+     * @var string
+     */
+    private $messageId;
+    /**
+     * @var string
+     */
+    private $replyTo;
+    /**
+     * @var string
+     */
+    private $correlationId;
+    /**
+     * @var array
+     */
+    private $headers;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getBody()
+    {
+        return $this->body;
+    }
+
+    /**
+     * @param string $body
+     */
+    public function setBody($body)
+    {
+        $this->body = $body;
+    }
```

Comment:
Ok, why not. In think we could use the original `AMQPEnvelope` class (as suggested by @Oliboy50) because it seems to already be a VO. But it is still coherent in regard of your whole PR 😃